pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,76 @@
-# Running a Full Node
+# Run a Full Node
+Running a full node allows your system to participate in a dYdX chain network. The recommended settings and best practices on this page help ensure your node stays healthy and up to date with the network.
 
-### Save your Chain ID in `dydxprotocold` config
+> Code snippets on this page use example values. Replace them with your own. See the [Network Configuration](../infrastructure_providers-network/network_constants.mdx) section of the documentation for network constants and other resources you need to configure a full node.
 
-Save the [chain-id](../infrastructure_providers-network/network_constants.mdx#chain-id). This will make it so you do not have to manually pass in the chain-id flag for every CLI command.
+## Prerequisites
+You need a running, non-validating full node that is connected to network. 
 
-```bash
-dydxprotocold config chain-id $CHAIN_ID
-```
-
-### Getting a Snapshot
+- If you created a system service for your node by following the instructions on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), start your node with the following command:
+  ```bash
+  stystemctl start dydxprotocol
+  ```
+- To start your node directly with Cosmovisor, you must include the flag `--non-validating-full-node=true`. Cosmovisor may prompt you to configure additional environment variables or include them in your command.
+  ```bash
+  cosmovisor run start --non-validating-full-node=true 
+  ```
 
-See [snapshot service](../infrastructure_providers-network/resources.mdx#snapshot-service).
+<!-- is this right? i also see 
+dydxprotocold start --p2p.seeds="..." --bridge-daemon-eth-rpc-endpoint="<eth rpc endpoint>" --non-validating-full-node=true -->
 
-### Starting a Full Node
+## Connect to Healthy Peers
+To keep your full node up to date, connect to healthy peers that have the latest state of the network. You can find a list of healthy peers with the latest state from the following services:
+- https://services.lavenderfive.com/mainnet/dydx#live-peers
```

Comment:
nit: this is for mainnet only. would be good to add an example for testnet, or link to another table to get values